playbook.yaml

```diff
@@ -55,14 +55,42 @@
         - objects/namespace-install.yaml
         - objects/argocd_server_route.yaml
 
-    - name: Patch ArgoCD admin psw
+    - name: Get Dex Service Account Token
+      shell: oc serviceaccounts get-token argocd-dex-server
```

Comment:
Can you use the openshift/ k8s module to get this? shell commands are an antipattern in  ansible

Reply:
hmmm I could, it turns into something more complicated but I'll take a look